components/Ink/README.md

```diff
@@ -8,7 +8,7 @@ path: /catalog/ink/
 api_doc_root: true
 -->
 
-<!-- This file was auto-generated using scripts/generate_readme Ink -->
+<!-- This file was auto-generated using ./scripts/generate_readme Ink -->
```

Comment:
Please revert this file.

Reply:
I am a bit confused. These changes were most of why I was rerunning the script.

I added this to the first comment to make that more clear

Reply:
If you think all of the comments should be formatted one way or another, I'd recommend that you update the script itself so it automatically normalizes the script name.  As far as I know, there's no reason why I couldn't call this script from the `catalog` directory (or any other subdirectory or parent directory) and generate a different script name during execution.

Reply:
The script is not hermetic in the output of this line unfortunately, so Rob is correct in the sense that it can generate different output based on how the script was invoked. 

That being said, `./scripts/generate_all_readmes` is meant to be our "canonical" way to invoke all of these scripts, so this PR is technically accurate in its standardization of the results.

I agree, though, that we should make a separate PR to fix the generate_readme script such that this generated line is consistent for different invocation styles.

Reply:
Filed https://github.com/material-components/material-components-ios/issues/8572 to track the follow-up work.